version/1.0.md

```diff
@@ -8,67 +8,62 @@ current: true
 
-## Open Science pillars are [the Company] culture
+## Open Science pillars as [the Company]'s culture
 
-Unless [the Company] has good reasons for not respecting them (such as National
-Security), every principle of the Open Science initiative should be considered
-as an organization's *credo*.
+Unless [the Company] has reasonable reasons (e.g., National Security concerns),
+every principle of the Open Science initiative should be treated as
+[the Company]'s *credo*.
 
-Each time [the Company] produces source code that could help other scientists in
-their everyday work (software, data analysis, web servers, etc.), the code
-should be open and stored in a public forge that eases community contributions
-and explicitly states on its licensing and credits.
+Each time [the Company] produces source code that may help other scientists in
+their everyday work (e.g., software, data analysis, web servers), the code
+should be made publicly available, allow for community contributions, and be
+clearly accompanied by any licensing information and/or credits.
 
-If relevant for the scientific community, scientific data collected by [the
-Company] should be stored in a public repository that explicitly states on its
-licensing and credits. [The Company] should carefully pay attention to the
-repository terms & conditions before submitting data to this repository.
-Institutional repositories (such as [Zenodo](https://zenodo.org)) must be the
-primary choice, but, third-party repositories can also be considered if the
-service maintainer (company or institution) explicitly states on the future of
-stored data when that maintainer status evolves (fusion, acquisition,
-liquidation).
+Any data collected by [the Company] that is relevant to the scientific community
+should be stored in a public repository, also explicitly accompanied by any
+licensing information and/or credits. [The Company] should pay careful attention
+to a repository's terms & conditions before submitting any data. Institutional
+repositories such as [Zenodo](https://zenodo.org) are preferred; however,
+third-party repositories may also be considered, provided that the service
+maintainer clearly addresses the treatment of stored data in the event that the
+maintainer's status were to change (e.g., fusion, acquisition, liquidation).
 
-When the time to publish research results has come, [the Company] submits its
-manuscript to Open Access journals with an Open Peer Review process.
+Once research results are ready for publication, [the Company] should submit its
+manuscript to Open Access journals using an open peer review process.
 
-## Users are not the product
+## User privacy is of utmost importance
 
-[The Company] must respect the user's right to privacy. [The Company] must not 
-sell or give away data about user's use patterns, locations, or other metadata
-without explicit, informed consent from the user.
+[The Company] must respect the user's right to privacy. Data about the user
```

Comment:
Should we talk about "users" rather than a singular "user"? I'm not terribly familiar with legalese, but the implication here is that there's just one user!